core/api/serializers.py

```
@@ -2,11 +1,208 @@
+        fields = '__all__'
+
+    def get_is_enrolled(self, obj):
+        request = self.context.get('request')
+        if request and request.user.is_authenticated:
+            return obj.enrollments.filter(
+                user=request.user,
+                payment_status='COMPLETED'
+            ).exists()
+        return False
+
+    def to_representation(self, instance):
```

Ruff (0.8.2)

Missing return type annotation for public function `to_representation`

(ANN201)